packages/react-data-grid/src/HeaderCell.js

```diff
@@ -51,8 +51,9 @@ class HeaderCell extends React.Component {
   };
 
   getWidthFromMouseEvent = (e: SyntheticMouseEvent): number => {
+    const node = ReactDOM.findDOMNode(this);
     let right = e.pageX || (e.touches && e.touches[0] && e.touches[0].pageX) || (e.changedTouches && e.changedTouches[e.changedTouches.length - 1].pageX);
-    let left = ReactDOM.findDOMNode(this).getBoundingClientRect().left;
+    let left = node ? node.getBoundingClientRect().left : 0;
```

Comment:
It will be cleaner to use `ref` instead of `ReactDOM.findDOMNode` and use `this.node` with a null check

```
render() {
...
   <div className={className} style={this.getStyle()} ref={(node) => { this.node= node; }}>
...
}
```

Thoughts?

Reply:
Yeah, that would be much better. Cheers